app/helpers/feedback_form_helper.rb

```diff
@@ -12,22 +12,14 @@ def show_feedback_form?
   end
 
   def show_quick_report?
-    if (params[:controller] == 'catalog' && params[:action] == 'show') or
+    (params[:controller] == 'catalog' && params[:action] == 'show') or
       (refered_from_catalog_show? &&
-        params[:controller] == 'feedback_forms' && params[:action] == 'new')
-      true
-    else
-      false
-    end
+        params[:controller] == 'feedback_forms' && params[:action] == 'new') || false
   end
 
   def refered_from_catalog_show?
     if request.referer.present?
-      if /(\/catalog\/|\/view\/)/.match?(request.referer)
-        true
-      else
-        false
-      end
+      /(\/catalog\/|\/view\/)/.match?(request.referer) || false
```

Comment:
Why do we need `|| false` here?

Reply:
It is what is in the code. I am not sure why it was done this way but it was.